### ISSUE TITLE
Add Time-based shadow 

### ISSUE THREAD

**Issue by MrAgentBlaze** [depressed-pho/natural-mystic-shaders#13]
like the sunrise the shadow faces the opposite like real life enabling the ability of sun clocks

**Comment by depressed-pho** [comment]
Yeah... that's what I really want to do, but sadly it's impossible now, due to a limitation of the current rendering system. The game doesn't consider the position of sun when it computes a light map.
